uaclient/entitlements/fips.py

```diff
@@ -342,11 +352,12 @@ def remove_packages(self) -> None:
 
     def _perform_enable(self, silent: bool = False) -> bool:
         if super()._perform_enable(silent=silent):
-            self.cfg.notice_file.try_remove(
-                "", messages.NOTICE_WRONG_FIPS_METAPACKAGE_ON_CLOUD
+            notices.remove(
+                self.cfg.root_mode,
+                NoticesDetails.WRONG_FIPS_METAPACKAGE_ON_CLOUD.value,
             )
-            self.cfg.notice_file.try_remove(
-                "", messages.FIPS_REBOOT_REQUIRED_MSG
+            notices.remove(
+                self.cfg.root_mode, NoticesDetails.FIPS_REBOOT_REQUIRED.value
```

Comment:
I'm confused by this one - @lucasmoura do you remember why this notice gets removed on enable?